add mobile hamburger drop-down menu

Besufekad-HAZ opened the pull request:
-Add hamburger menu
-Add mobile drop-down menu
-Fix stylint errors with no descending specificity problem

AbayomiOlaoye commented on index.js:
```diff
@@ -1 +1,25 @@
-// JS code goes here
\ No newline at end of file
+// JS code goes here
+const hamburger = document.querySelector('.hamburger')
+const mobnav = document.querySelector('.mob__nav__menu')
+const nvsection = document.querySelector('.nv__section')
+const hamburger2 = document.querySelector('.mob__nav__menu .hamburger2')
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active')
+  mobnav.classList.toggle('active')
+  nvsection.classList.toggle('active')
+})
+
+document.querySelectorAll('.nv__item').forEach((n) =>
+  n.addEventListener('click', () => {
+    hamburger.classList.remove('active')
+    hamburger.style.display = 'block'
+    mobnav.classList.toggle('active')
+  })
+)
+
+hamburger2.addEventListener('click', () => {
+  hamburger.style.display = 'block'
+  hamburger.classList.remove('active')
+  mobnav.classList.toggle('active')
+})
```
- [ ] Great job 👍 everything checks out well but I think you should try to use `camelCasing` convention in your variable name declaration when you are joining 2 or more words together. This is best practice and it allows more clarity when reading code. Examples of where you could use this are found in `mobnav` which could have been `mobNav` and `nvsection` as `nvSection` 😉 . This will increase the understandability of your code such you won't have issues remembering what is what.

AbayomiOlaoye commented on index.js:
```diff
@@ -1 +1,25 @@
+  nvsection.classList.toggle('active')
+})
+
+document.querySelectorAll('.nv__item').forEach((n) =>
+  n.addEventListener('click', () => {
```
- [ ] You did a great job with this implementation, and to completely make it the masterpiece that I think it currently is, kindly avoid the use of arbitrary or generic names. As professionals with the mindset of maintainability or collaboration, it is best to use descriptive naming techniques for our `variables`, `parameter` or `arguments` in functions. For this, do you think you can use a more descriptive word like `item`, `link` or `nav` instead of the generic `n` that you have here? This will make your code more readable and make working with others less stressful. 💪

AbayomiOlaoye reviewed (changes requested):
## STATUS: CHANGE REQUIRED ♻️ 

Hi @Besufekad-HAZ :wave, 

Good job with this milestone so far!
You have done a brilliant job in this **PR but there are just a few more issues to work on before moving on to the next project - but you are almost there you know!

![so_close](https://user-images.githubusercontent.com/99715640/211950260-0e3881c0-9d4f-4ea2-a670-9ef45a23a0f5.gif)

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Awesome ReadMe ✔️ 
- Great project implementation ✔️ 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

addod19 reviewed (approved):
### Hi @Besufekad-HAZ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job addressing the comments by the previous code reviewer :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Besufekad-HAZ commented:
Thank you for checking the changes and Approving them...🙏🙏🙏

On Tue, Jan 31, 2023 at 11:10 AM Daniel Larbi Addo ***@***.***>
wrote:

> ***@***.**** approved this pull request.
> Hi @Besufekad-HAZ <https://github.com/Besufekad-HAZ>,
>
> Your project is complete! There is nothing else to say other than... it's
> time to merge it [image: :shipit:]
> Congratulations! 🎉
> Highlights
>
>    - Nice job addressing the comments by the previous code reviewer 🚀
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Besufekad-HAZ/My_Portfolio/pull/9#pullrequestreview-1276664703>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFSK2MJWYWBRSLNTDMXHFWDWVDCGPANCNFSM6AAAAAAULSK6MI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
